modules/electrophysiology_browser/php/sessions.class.inc

```diff
@@ -48,9 +48,10 @@ class Sessions extends \NDB_Page
     function _hasAccess(\User $user) : bool
     {
         return ($user->hasPermission('electrophysiology_browser_view_allsites')
-                || ($user->hasCenter($this->timepoint->getCenterID())
+                || ($user->hasStudySite()
+                || $user->hasCenter($this->timepoint->getCenterID())
                 && $user->hasPermission('electrophysiology_browser_view_site')
-            )
+                    )
```

Comment:
too much whitespace here i think

Reply:
I found it better indicates where the beginning parenthesis starts since they're vertically aligned. Will undo it.